packages/next/README.md

```diff
@@ -1110,9 +1110,7 @@ export default class MyDocument extends Document {
   render() {
     return (
       <html>
-        <Head>
-          <style>{`body { margin: 0 } /* custom! */`}</style>
-        </Head>
+        <Head />
```

Comment:
I'm not sure if this shows that `<Head>` is required. Maybe adding some note below marked as **Important**, noting that since you're extending the base `Document` class the following components are still required: `Head`, `Main`, `NextScripts` etc.